test/weighted_matching_test.cpp

```diff
@@ -147,8 +149,8 @@ int test_main(int, char*[])
     // you can also lower the max_num_v and/or max_num_e
     
     std::size_t max_num_v = 16;
-    std::size_t max_num_e = 20;
-    std::size_t batch_size = 100;
+    std::size_t max_num_e = 24;
+    std::size_t batch_size = 16;
```

Comment:
How do you calculate the optimal test size?

Reply:
I didn't, because I am using a brute-force verifier, it can be time-consuming when edge num increases (exponential), and travis-ci has time limit

Reply:
Why not optimize it locally first?

Reply:
I can wait for as long as it takes on my machine, but I didn't know about travis-ci time limit

Reply:
I'm sure we can increase the time limit if it increases correctness.  But you should be able to determine the rough upper and lower bounds of these variables for testing correctness.

Reply:
You're right, I will give a rough time estimate regarding edge num